spec/index.html

```diff
@@ -315,31 +315,30 @@ <h3>RDF Vocabularies and Namespace IRIs</h3>
     <h3>Triple Terms and Reification</h3>
 
     <p>A <a>triple term</a> is an [=RDF triple=] used as an [=RDF term=]
-      in the [=object=] position of another triple.</p>
-
-    <p>A <a>triple term</a> is not necessarily asserted, allowing
-      statements to be made about other statements that may not be
-      asserted within an <a>RDF graph</a>.
-      This allows statements to be made about relationships that may be contradictory.
-      For a <a>triple term</a> to be asserted in a graph,
-      it must appear as an <a>asserted triple</a> in that same graph.</p>
+      in another triple. This use is a reference to a <a>proposition</a>. For
+      the triple term to also be asserted, it must appear in a graph as an <a>asserted
+      triple</a>. This allows for statements to be made about statements
+      independent of their assertion within an <a>RDF graph</a>.
+    </p>
 
     <p>RDF terms that appear in a triple term have the same denotation as when they appear in an asserted triple in the graph. 
```

Comment:
```suggestion
    <p>RDF terms that [=appear=] in a triple term have the same denotation as when they [=appear=] in an asserted triple in the graph. 
```

Reply:
Or we mark them all?
```suggestion
    <p>[=RDF terms=] that [=appear=] in a [=triple term=] have the same
      <a data-lt="denotes">denotation</a> as when they appear in an
      [=asserted triple=] in the <a data-lt="RDF graph">graph</a>.
      For this reason, we say that triple terms are <i>transparent</i>.</p>
```
(Do we also want to `<dfn>transparent</dfn>`?)

Reply:
I'm of very mixed mind on this. Sometimes linking every occurrence of (one, some, many, all) term(s) makes sense. Sometimes "first occurrence" on a page is the rule (typically for output on paper). Sometimes "first occurrence" in a paragraph (but then what about when paragraphs are short, one-liners? or huge, 30-liners? and what about list entries? among other neighboring cases...).

W3C Style Guide is noncommittal on this, suggesting that "[at least the first mention of a source](https://www.w3.org/guide/manual-of-style/#citation)" should be linked. It says nothing about how much additional linkage is recommended or desirable.

My rule of thumb is that links should happen sufficiently often to be conveniently reached from every instance of the linked term. Often that means linking every instance. This also has the benefit of being explicit that each linked instance has the same meaning, where an unlinked instance *might* be read as having a different meaning.

All of which brings me to *generally* linking every instance.

Reply:
I followed the pattern I see in this document (at least within section 1), which links first occurrence of each paragraph (at most). I thus don't think marking up the same term within the same sentence is useful (to me it raises the question if any non-linked use of a term is less specific). If paragraphs are long, that might warrant repetitive linking.